README.md

```diff
@@ -143,6 +143,14 @@ source simulation_ws/install/local_setup.sh
 roslaunch cloudwatch_simulation worldforge_turtlebot_navigation.launch
 ```
 
+By default, WorldForge packages will load the exported world. To override, specify the environment variable `WORLD_ID`. 
+
+```bash
+# use worldId found in "src/aws_robomaker_worldforge_worlds/worlds"
+# e.g, generation_05wq8sybdcn2_world_1
+export WORLD_ID=<worldId>  
+```
+
```

Comment:
This part belongs to `Running simulation with a WorldForge world`. However, since map generation now is NOT optional, the prerequisite step of map generation will be merged in here so this part could be removed.